modules/opp-architecture-installation.adoc

```diff
@@ -2,14 +2,25 @@
 //
 // * architecture/opp-architecture.adoc
 
-:_module-type: PROCEDURE
+:_module-type: CONCEPT
 [id="opp-architecture-installation_{context}"]
 = Install {product-title} products
 
-To install {product-title}, you must install two products in a specific order. Install {ocp} first. Then, install {rh-rhacm}. It does not matter what order you install the remaining products: {quay}, {rh-storage-essentials-first}, and {acs}. See the following detailed installation information for each product:
+To install {product-title}, you must install {ocp} followed by {rh-rhacm}. The additional products are installed by applying an {rh-rhacm} PolicySet: {quay}, {rh-storage-essentials-first}, and {acs}. 
 
-* {ocp} - link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-preparing#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
-* {rh-rhacm} - link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing[Installing {rh-rhacm}]
-* {acs} - link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/installing/installing-rhacs-on-red-hat-openshift#install-rhacs-ocp[Installing {acs-short} on Red Hat OpenShift]
-* {rh-storage-essentials-first} - link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12[Deploying {rh-storage-data-foundation}]
-* {quay} - link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.8/html/deploy_red_hat_quay_on_openshift_with_the_quay_operator/index[Deploy {quay} on OpenShift with the Quay Operator]
+.Prerequisites
+
+* Install {ocp} 4.10 or later.
+* Install {rh-rhacm} 2.7 or later.
+* Ensure that the placement bindings match the namespace for the hub and other managed clusters.
++
+[source,yaml]
+
+----
```

Comment:
This yaml can be removed if you are ok referencing the setup instructions in the previous comment.